documentation/IDTA-01001/modules/ROOT/IDTA-01001.adoc

```diff
@@ -3634,16 +3642,19 @@ _Identifier_
 
 
 ====
-Note: for any string conformant to link:https://datatracker.ietf.org/doc/html/rfc8089[RFC8089], the "file" URI scheme (for relative and absolute file paths) applies.
+Note: URL, based on [RFC-3986](https://www.rfc-editor.org/rfc/rfc3987) /[RFC-3987](https://www.rfc-editor.org/rfc/rfc3987), but more recently defined as living standard by WHATWG, as suggested by W3C: https://url.spec.whatwg.org/
 ====
 
 
 a|
-./Specification.pdf
+\./Specification.pdf
```

Comment:
What's the reason for the backslash `\` here? As far as I know, only Windows machines allow for backslashes in directory paths.

Reply:
otherwise the . is not displayed in asciidoc - at least this was this way in my editor

Reply:
\ removed, thank you